.github/workflows/algolia-scraper.yml

```diff
@@ -0,0 +1,25 @@
+name: algolia-scraper
+on:
+  push: # only run on push events
+    branches:
+      - main
+  check_suite: # only run check suite events
+    types: [completed]
+  workflow_dispatch: # only run on workflow dispatch events
+jobs:
+  scrape:
+    runs-on: ubuntu-latest # specifies the OS image to use for the job
+    steps: # specifies the commands to run
+      - name: check out code 🛎
```

Comment:
Please follow the same style as we're using in https://github.com/Seneca-ICTOER/IPC144/pull/48/files#diff-259f2188de53828dcb003900d2a84cfd00a909870115a6e14ddc019e96255087R11, especially adding the `cache` for `yarn` so this goes faster.

Reply:
ok